packages/react-native-otlp/ios/RNEmbraceOTLP.swift

```diff
@@ -147,7 +151,21 @@ class RNEmbraceOTLP: NSObject {
                         crashReporter = EmbraceCrashReporter()
                     }
 
-                    let servicesBuilder = CaptureServiceBuilder().addDefaults()
+                    let servicesBuilder = CaptureServiceBuilder()
+                    
+                    let urlSessionServiceOptions = URLSessionCaptureService.Options(
+                        // allowing to enable/disable NSF by code
+                        injectTracingHeader: !config.disableNetworkSpanForwarding,
```

Comment:
added the ability of enable/disable NSF even when the full implementation is not part of this PR